test/IRGen/section_structs.swift

```diff
@@ -38,8 +38,14 @@ struct MyStruct5 {
 }
 @_section("__TEXT,__mysection") var g_MyStruct5: MyStruct5 = MyStruct5(q: MyStruct4(a: 42, s: MyStruct1(a: 43, b: 44)), r: MyStruct4(a: 42, s: MyStruct1(a: 43, b: 44)))
 
+@_section("__TEXT,__mysection") let utf8OfStaticString = StaticString("hello").utf8Start
```

Comment:
@eeckstein, strangely this broke on the 32-bit Android armv7 CI but not on the aarch64 CI:
```
/home/ubuntu/jenkins/workspace/oss-swift-RA-linux-ubuntu-16.04-android/swift/test/IRGen/section_structs.swift:41:37: error: global variable must be a compile-time constant to use @_section attribute
@_section("__TEXT,__mysection") let utf8OfStaticString = StaticString("hello").utf8Start
                                    ^
```
This compiler on the Android CI is built with a full bootstrap on linux, as that CI has no prebuilt Swift compiler installed, so the swift-syntax parser is not built. Maybe that has something to do with it, but weird that it only hits for 32-bit compilation.

Reply:
@finagolfin do you have any error log or a link to the ci failure?

Reply:
Sorry, [copied that but forgot to paste it](https://ci-external.swift.org/job/oss-swift-RA-linux-ubuntu-16.04-android/6157/consoleText).

Reply:
Let's disable the test for 32-bit

Reply:
SGTM

Reply:
https://github.com/apple/swift/pull/70540